lightning/src/blinded_path/message.rs

```diff
@@ -52,12 +54,27 @@ pub(crate) struct ForwardTlvs {
 
 /// Similar to [`ForwardTlvs`], but these TLVs are for the final node.
 pub(crate) struct ReceiveTlvs {
-	/// If `path_id` is `Some`, it is used to identify the blinded path that this onion message is
-	/// sending to. This is useful for receivers to check that said blinded path is being used in
-	/// the right context.
```

Comment:
Let's keep this comment.

Reply:
Done! [6ff2e49](https://github.com/lightningdevkit/rust-lightning/commit/6ff2e49123c3135dccea720d7104e6ad5b8f7308)
Thanks!